### ISSUE TITLE
Inherited base model schema not being included in final output file

### ISSUE THREAD

**Issue by NeilJ247** [WindomZ/swagger-merger#55]
Hi,

I have the following:
path.yml

```
get:
    tags:
    - "Tag One"
    responses:
      '200':
          description: Just an example of includes
          content:
              application/json:
                  schema:
                      type: array
                      items:
                          $ref: './schema.yml#/put'
```

schema.yml:

```
base:
    type: object
    properties:
        propOne:
            type: string
            example: "Example String"
        propTwo:
            type: integer
            example: 0

put:
    allOf:
    - $ref: '#/base'
    - type: object
    properties:
        undelete:
            type: boolean
            example: false
        reprocess:
            type: boolean
            example: false
```
The merger doesn't pull in the base properties which results in the output file referencing something that doesn't exist:

```
  "paths": {
    "/some/path": {
      "get": {
        "tags": [
          "Tag One"
        ],
        "responses": {
          "200": {
            "description": "Just an example of includes",
            "content": {
              "application/json": {
                "schema": {
                  "type": "array",
                  "items": {
                    "allOf": [
                      {
                        "$ref": "#/base"
                      },
                      {
                        "type": "object"
                      }
                    ],
                    "properties": {
                      "undelete": {
                        "type": "boolean",
                        "example": false
                      },
                      "reprocess": {
                        "type": "boolean",
                        "example": false
                      }
                    }
                  }
                }
              }
            }
          }
        }
      }
    }
  }
```
I've included the example files to help solve this issues.

Thanks.

[example.zip](https://github.com/WindomZ/swagger-merger/files/2505459/example.zip)

**Comment by darkantoine** [comment]
Got the same issue as well!

**Comment by rulrok** [comment]
I was just going to open a PR about this.
I does not resolve the refs recursively!
Wow, this issue is old.

**Comment by WindomZ** [comment]
```
{
  "openapi": "3.0.0",
  "info": {
    "title": "Example",
    "version": "1.0"
  },
  "tags": [
    {
      "name": "Tag One"
    }
  ],
  "paths": {
    "/some/path": {
      "get": {
        "tags": [
          "Tag One"
        ],
        "responses": {
          "200": {
            "description": "Just an example of includes",
            "content": {
              "application/json": {
                "schema": {
                  "type": "array",
                  "items": {
                    "allOf": [
                      {
                        "type": "object",
                        "properties": {
                          "propOne": {
                            "type": "string",
                            "example": "Example String"
                          },
                          "propTwo": {
                            "type": "integer",
                            "example": 0
                          }
                        }
                      },
                      {
                        "type": "object"
                      }
                    ],
                    "properties": {
                      "undelete": {
                        "type": "boolean",
                        "example": false
                      },
                      "reprocess": {
                        "type": "boolean",
                        "example": false
                      }
                    }
                  }
                }
              }
            }
          }
        }
      }
    }
  }
}
```

I think that's what you expect, right? @NeilJ247

**Comment by WindomZ** [comment]
Issue closed in version v1.5.1.
